src/app/modules/shared-components/shared-components.module.ts

```diff
@@ -160,6 +162,7 @@ import { ErrorComponent } from './components/error/error.component';
     AccessCodeViewComponent,
     LanguagePickerComponent,
     ErrorComponent,
+    LoadingComponent,
```

Comment:
If we have an app LoadingComponent, we should forbid usage of ProgressSpinner now, that means:
- Removing the exports or ProgressSpinner in SharedComponentsModule
- Removing declarations of ProgressSpinner in the other modules

Reply:
Fixed